mne/preprocessing/ica.py

```diff
@@ -137,6 +137,20 @@ class ICA(ContainsMixin):
               Extended-Infomax seems to be more stable in this respect
               enhancing reproducibility and stability of results.
 
+    .. warning:: ICA is sensitive to low-frequency drifts and therefore
+                 requires the data to be high-pass filtered prior to fitting.
+                 Typically, a cutoff frequency of 1 Hz is recommended. Note
+                 that FIR filters prior to MNE 0.15 used the 'firwin2' design
+                 method, which generally produces rather shallow filters that
+                 might not work for ICA processing. Therefore, it is
+                 recommended to use IIR filters for MNE up to 0.14. In MNE
+                 0.15, FIR filters can be designed with the `firwin` method,
```

Comment:
double-backticks. And you use quotes above, might as well be consistent while you're changing it